packages/console/src/command-history/CommandHistory.tsx

```diff
@@ -351,6 +354,10 @@ class CommandHistory extends Component<
     }
   }
 
+  update(): void {
```

Comment:
I'd prefer this to be named something else. It's not an update, it's on show or scroll to bottom